### PR TITLE
Bld 24506

### PR DESCRIPTION
**Pull request by ozbenoved**

### CODE REVIEW

File: SDKSamples/app/src/main/java/com/sdk/samples/MainActivity.kt [hunk 1]
```diff
@@ -51,6 +47,10 @@ class MainActivity : AppCompatActivity() {
                 "com.sdk.sample.action.BOLD_CHAT_UPLOAD",
                 "Custom upload on live chat",
                 resources.getDrawable(R.drawable.outline_publish_black_24)
+            ), SampleTopic(
+                "com.sdk.sample.action.BOLD_UPLOAD_NO_UI",
```
Comment (tehilaR2) [inline SDKSamples/app/src/main/java/com/sdk/samples/MainActivity.kt]: keep it synced with the other upload action: "BOLD_CHAT_UPLOAD_NO_UI"

File: SDKSamples/app/src/main/java/com/sdk/samples/MainActivity.kt [hunk 2]
```diff
@@ -51,6 +47,10 @@ class MainActivity : AppCompatActivity() {
                 "com.sdk.sample.action.BOLD_CHAT_UPLOAD",
                 "Custom upload on live chat",
```
Comment (tehilaR2) [inline SDKSamples/app/src/main/java/com/sdk/samples/MainActivity.kt]: please add this to the Strings resources

File: SDKSamples/app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -1,17 +1,18 @@
...
     <string name="start_chat">Start Chat</string>
     <string name="department_selection"><u><b>Select a department for availability check</b></u></string>
     <string name="start_live_no_dep_instruction">Start live chat or check availability without preconfigured department</string>
     <string name="chat_with_bot">Chat with bot ai</string>
     <string name="chat_with_bold">Chat with live agent</string>
-    <string name="chat_with_async">Async chat with live agent</string>
     <string name="bot_chat_with_welcome">Chat with ai: customizing welcome message</string>
     <string name="standalone_autocomplete">Standalone Autocomplete</string>
     <string name="bot_to_bold_with_prechat">Live chat escalation from ai &amp; Prechat values</string>
+    <string name="bold_upload_without_ui">Bold upload without UI</string>
```
Comment (tehilaR2) [inline SDKSamples/app/src/main/res/values/strings.xml]: "Uploading with live uploader"

File: SDKSamples/app/src/main/java/com/sdk/samples/MainActivity.kt (hunk 1 above)
Comment (ozbenoved) [reply SDKSamples/app/src/main/java/com/sdk/samples/MainActivity.kt]: ok

File: SDKSamples/app/src/main/java/com/sdk/samples/MainActivity.kt (hunk 2 above)
Comment (ozbenoved) [reply SDKSamples/app/src/main/java/com/sdk/samples/MainActivity.kt]: ok

File: SDKSamples/app/src/main/res/values/strings.xml (hunk 3 above)
Comment (ozbenoved) [reply SDKSamples/app/src/main/res/values/strings.xml]: ok

**Pull request merged**